examples/purchase-tester/src/main/res/layout/package_card.xml

```diff
@@ -111,7 +116,7 @@
                     android:paddingHorizontal="4dp"
                     android:text="@string/subscription_options"
                     android:textStyle="bold"
-                    android:visibility="@{isSubscription ? View.VISIBLE : View.GONE}"
+                    android:visibility="@{isSubscription &amp; isPlayStore ? View.VISIBLE : View.GONE}"
```

Comment:
SHouldn't this be using `&&` instead of `&amp;` `"@{isSubscription && isPlayStore ? View.VISIBLE : View.GONE}`